### PR TITLE
Various tweaks, fixes, UI components used in meds tracking 

### PR DESCRIPTION
**Pull request by syoung-smallwisdom**

### CODE REVIEW

**Review by DanWebster — APPROVED** [review]
Haven't tested this on the Catalog, but on code-level review, all looks good to me

File: Research/ResearchUI/iOS/Views/RSDRoundedToggleButton.swift [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        heightConstraint.isActive = true
+        
+        let widthConstraint = self.widthAnchor.constraint(equalToConstant: RSDRoundedButton.defaultWidthWith2Buttons)
+        widthConstraint.priority = .defaultLow
+        widthConstraint.isActive = true
+        
+        let image = UIImage(named: "checkmark", in: Bundle(for: RSDRoundedButton.self), compatibleWith: self.traitCollection)
+        setImage(image, for: .selected)
+        
+        self.imageEdgeInsets = UIEdgeInsets(top: 10, left: -10, bottom: 10, right: 10)
```
Comment (Erin-Mounts) [inline Research/ResearchUI/iOS/Views/RSDRoundedToggleButton.swift:136]: is -10 intentional for the left edge inset?

**Review by Erin-Mounts — APPROVED** [review]
One question, otherwise looks good

File: Research/ResearchUI/iOS/Views/RSDRoundedToggleButton.swift (hunk 1 above)
Comment (syoung-smallwisdom) [reply Research/ResearchUI/iOS/Views/RSDRoundedToggleButton.swift:136]: Yes. It's weird, but that's the UI tweet that makes the checkbox not overlay the text. ¯\_(ツ)_/¯

**Pull request merged**